src/LimitingPaymaster.sol

```diff
@@ -19,6 +19,9 @@ import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
 contract LimitingPaymaster is BasePaymaster {
     using UserOperationLib for UserOperation;
 
+    uint256 public constant newSenderPostOpOverhead = 23231;
```

Comment:
Can you point me to the characterization output that determined these gas ops?

Reply:
we ran a previous version onchain and logged the diffs, updating these constants until they matched the overhead exactly